registration/forms.py

```diff
@@ -355,6 +385,41 @@ def update_user(self, application, commit=True):
                 application.user.profile.save()
                 application.save()
 
+        if self.restart_fields_changed():
+            messages.add_message(self.request, messages.INFO,
+                                 "Thank you for submitting these changes - we will rebuild your instance to "
+                                 "apply them, and email you to confirm once it is up to date.")
```

Comment:
I noticed that the "Thank you for applying for the OpenCraft beta. We will contact you shortly" message still shows up when submitting changes as well. The reason for this is that this "Thank you" message shows up whenever the [application is pending](https://github.com/open-craft/opencraft/blob/d891795e7bf7a568222331859403e9584dbf22c3/registration/templates/registration/registration.html#L27-L28), so I don't feel it's that weird.
![](https://i.imgur.com/0IyKeC4.png)

Reply:
Yes, this was a conscious decision, one message is for the changes, the other for the status. Both apply. To clarify, to could prepend „Application status: “ to the 2nd one. But I don't think it's needed.

Reply:
@clemente Yes, that seem fine. Maybe just remove that exclamation mark - from both actually. It looks like something went wrong, when it's just informative.

Reply:
I removed the exclamation marks.